solutions/src/test/java/ru/sberbank/school/task08/ManagerTest.java

```diff
@@ -0,0 +1,11 @@
+package ru.sberbank.school.task08;
+
+public abstract class ManagerTest {
+
+    protected static final String PATH =
+        "D:\\dev\\java\\Test";
+    protected static final String FILENAME = "file.txt";
+
+    abstract void saveEmptyGameObjects();
+    abstract void saveAndLoadGame() throws SaveGameException;
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a122155572>*